projects/kit/components/input-date-multi/input-date-multi.directive.ts

```diff
@@ -0,0 +1,34 @@
+import {Directive} from '@angular/core';
+import {TuiDay} from '@taiga-ui/cdk';
+import {AbstractTuiTextfieldHost, tuiAsTextfieldHost} from '@taiga-ui/core';
+import {tuiImmutableUpdateInputDateMulti} from '@taiga-ui/kit/utils';
+
+import {TuiInputDateMultiComponent} from './input-date-multi.component';
+
+@Directive({
+    selector: 'tui-input-date[multiple]',
+    providers: [tuiAsTextfieldHost(TuiInputDateMultiDirective)],
+})
+export class TuiInputDateMultiDirective extends AbstractTuiTextfieldHost<TuiInputDateMultiComponent> {
```

Comment:
This directive is no needed, it is only used in native inputs and we decided not to implement native input for now.

Reply:
fixed